app/javascript/modules/listingDetails/ListingDetailsHMITable.tsx

```diff
@@ -216,28 +235,7 @@ export const ListingDetailsHMITable = ({ listing }: ListingDetailsEligibilityPro
   }
 
   return (
-    <ListSection
```

Comment:
I like that you pulled this out of the component, which is more in line with how other components are structured in the Eligibility section 👍